hypervisor/include/common/irq.h

```diff
@@ -14,14 +14,18 @@
 
 typedef void (*irq_action_t)(uint32_t irq, void *priv_data);
 
-/* any field change in below required irq_lock protection with irqsave */
+/**
+ * @brief interrupt descritpor
+ * 
```

Comment:
Extra space to be removed at the end of the line
```suggestion
 *
```